packages/react-native-editor/package.json

```diff
@@ -153,39 +52,82 @@
     "js-beautify": "^1.7.5",
     "jsc-android": "^241213.1.0",
     "jsdom-jscore-rn": "git+https://github.com/iamcco/jsdom-jscore-rn.git#a562f3d57c27c13e5bfc8cf82d496e69a3ba2800",
-    "jsx-to-string": "^1.3.1",
-    "memize": "^1.0.5",
     "metro-react-native-babel-transformer": "^0.53.0",
-    "moment": "^2.22.1",
-    "moment-timezone": "^0.5.16",
     "node-libs-react-native": "^1.0.2",
-    "node-sass": "^4.12.0",
-    "react": "16.8.6",
-    "react-native": "jtreanor/react-native#v0.60.0-patched",
     "react-native-dark-mode": "git+https://github.com/wordpress-mobile/react-native-dark-mode.git#f09bf1480e7b34536413ab3300f29e4375edb2c6",
     "react-native-hr": "git+https://github.com/Riglerr/react-native-hr.git#2d01a5cf77212d100e8b99e0310cce5234f977b3",
     "react-native-keyboard-aware-scroll-view": "git+https://github.com/wordpress-mobile/react-native-keyboard-aware-scroll-view.git#gb-v0.8.7",
     "react-native-modal": "^6.5.0",
     "react-native-recyclerview-list": "git+https://github.com/wordpress-mobile/react-native-recyclerview-list.git#eadaa2f62d2f488d4dc80f9148e52b62047297ab",
     "react-native-safe-area": "^0.5.0",
     "react-native-svg": "git+https://github.com/wordpress-mobile/react-native-svg.git#d466eb06a38e6f553f08666f0ca784d47fd6f724",
-    "react-native-video": "git+https://github.com/wordpress-mobile/react-native-video.git#ad64098edc4ce1cd0ad783cb1843df634d997fdb",
-    "react-redux": "^5.0.7",
-    "redux": "^3.7.2",
-    "redux-multi": "^0.1.12",
-    "redux-optimist": "^1.0.0",
-    "refx": "^3.0.0",
-    "rememo": "^3.0.0",
-    "shallowequal": "^1.0.2",
-    "showdown": "^1.8.6",
-    "simple-html-tokenizer": "^0.4.1",
-    "tannin": "^1.0.1",
-    "tinycolor2": "^1.4.1",
-    "traverse": "^0.6.6",
-    "turbo-combine-reducers": "^1.0.2",
-    "underscore": "^1.9.1"
+    "react-native-video": "git+https://github.com/wordpress-mobile/react-native-video.git#ad64098edc4ce1cd0ad783cb1843df634d997fdb"
+  },
+  "publishConfig": {
+    "access": "public"
   },
-  "resolutions": {
-    "@react-native-community/cli": "^1.5.2"
+  "scripts": {
```

Comment:
A similar question, how do you plan to work with this package given that there is a huge number of scripts? Some of them reference the local `node_modules` folder but others go two levels up.

Reply:
I'm not sure yet, that's something we can discuss here or on follow up PRs when we tackle this problem. I think that given the amount of scripts we might want to keep those here for now as to not clutter the main `package.json`.

Reply:
> Some of them reference the local node_modules folder but others go two levels up.

In theory there should not be any local `node_modules` folder, since that indicates a conflict. We'll work to fix those conflicts.